.env.prod

```
@@ -0,0 +1,5 @@
+DATABASE_URL=postgresql://postgres:postgres@db:5432/book_store?schema=public
+BACKEND_API_URL=http://localhost:3000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BACKEND_API_URL key should go before the DATABASE_URL key

(UnorderedKey)